.storybook/main.ts

```diff
@@ -5,15 +5,21 @@ const config: StorybookConfig = {
     name: '@storybook/nextjs',
     options: {},
   },
+
   stories: ['../src/**/*.@(mdx|stories.@(js|jsx|ts|tsx))'],
+
   addons: [
     '@storybook/addon-links',
     '@storybook/addon-essentials',
     'storybook-dark-mode',
   ],
+
   staticDirs: ['../public'],
-  docs: {
-    autodocs: true,
+
+  docs: {},
+
+  typescript: {
```

Comment:
See : https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#react-docgen-component-analysis-by-default

According to what is said, I think it is better to keep the full fidelity of react-docgen, that is why I left this.